shared/actions/listActions.js

```diff
@@ -99,17 +99,21 @@ export const getDetailImage = (key) => {
     queries.push(createQueryObject('item', key));
     // const address = `${HTTP}${'SERVER_ADDR'}${ENDPOINT_IMAGE}${queryBuilder(queries)}`;
     // const address = `https://goober.herokuapp.com/api/images/${queryBuilder(queries)}`;
-    getAccessToken().then((accessToken) => {
+    return getAccessToken().then((accessToken) => {
       const address = `${HTTPS}${SERVER_ADDR}${ENDPOINT_IMAGE}/${queryBuilder(queries)}`;
       const headers = getAuthHeaders(accessToken);
       return fetch(address, {
         method: 'GET',
         headers
       })
       .then(response => response.json())
-      .then(json =>
-        dispatch(receiveImages(json))
-      );
+      .then(json => {
+        return new Promise((resolve) => {
+          dispatch(receiveImages(json));
+          resolve(json.values);
```

Comment:
You use this function's return variable at line 121 of `components/activityCard.js`
And it was named json.
So i wonder that do you use json or json.value?
If you use json.value, please rename the variable at line 122 of `components/activityCard.js`